docs/_static/auth/iam.yaml

```diff
@@ -406,6 +406,42 @@ paths:
           description: |
             This is returned when an OAuth2 Client is unauthorized to talk to the introspection endpoint.
             Note: introspection of an invalid or malformed token returns a 200 where with field 'active'=false
+  /iam/{id}/status/{page}:
+    parameters:
+      - name: id
+        in: path
+        required: true
+        description: the id part of the web DID
+        schema:
+          type: string
+          example: EwVMYK2ugaMvRHUbGFBhuyF423JuNQbtpes35eHhkQic
+      - name: page
+        in: path
+        required: true
+        description: statuslist credential id
+        schema:
+          type: string
+          example: 1
+    get:
+      tags:
+        - oauth2
```

Comment:
I don't think this should have the OAuth2 tag. In Java, it generates a service class for each tag. So you'd get an `OAuth2Service` class which includes this operation (but it's not OAuth2)